src/Landing/containers/Landing.jsx

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable react/destructuring-assignment */
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import '../CSS/Landing.scss';
+import RecommendedArticles from '../components/RecommendArticles';
+import LatestArticles from '../components/LatestArticles';
+import fetchLatestArticles from '../redux/actions/latestArticlesAction';
+import Loading from '../../commons/components/Loading';
+
+export class Landing extends Component {
+  componentWillMount() {
+    this.props.fetchLatestArticles();
+  }
+
+  render() {
+    const {
+      latestArticles,
+      recommendedArticles,
+      fetchStatus,
+      message,
+    } = this.props;
+
+    return (
+      <div className="landing">
+        {
+        message === 'No Articles'
```

Comment:
Use the fetch status in your redux store instead.

Reply:
@AGMETEOR I am using status from the store here though it's value is assigned to fetchStatus as you may see below.
<img width="530" alt="Screenshot 2019-07-17 at 10 40 45" src="https://user-images.githubusercontent.com/21230051/61356709-983da300-a87f-11e9-8086-90b755a656cb.png">

Reply:
@alexxsanya `fetchStatus` should then be the one used to make the check that decides when to show the loader

Reply:
This has been worked on. thanks